packages/api/src/graphql/mutation.private.ts

```diff
@@ -644,12 +644,17 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
     createArticle: {
       type: GraphQLNonNull(GraphQLArticle),
       args: {input: {type: GraphQLNonNull(GraphQLArticleInput)}},
-      async resolve(root, {input}, {authenticate, dbAdapter}) {
-        const {roles} = authenticate()
+      async resolve(root, {input}, {authenticateUser, dbAdapter}) {
+        const {
+          user: {id},
+          roles
+        } = authenticateUser()
         authorise(CanCreateArticle, roles)
-
+        const user = await dbAdapter.user.getUserByID(id)
```

Comment:
This shouldn't happen here. This should happen in the editor when the user clicks on "New Article"